devel/namespaces/index.md

```diff
@@ -86,36 +82,51 @@ Most roles are defined by some kind of automatic usage in a TaskCluster componen
 However, some are defined by convention.
 Both are listed here:
 
-* `assume:client-id:<clientId>` -
+* `client-id:<clientId>` -
    Roles with this prefix give the scopes for client credentials.
+   In general, scopes should be assigned directly to clients, instead.
 
-* `assume:hook-id:<hookGroupId>/<hookId>` -
+* `hook-id:<hookGroupId>/<hookId>` -
    Roles of this form give the scopes used to create tasks on behalf of hooks.
 
-* `assume:mozilla-group:<groupName>` -
+* `moz-tree:level:<level>` -
+   Roles of this form include the basic scopes available to version-control trees at each of the three Mozilla source-code managament levels.
+   They are useful as shorthand to configure `repo:*` roles.
+   See [Mozilla Commit Access Policy](https://www.mozilla.org/en-US/about/governance/policies/commit/access-policy/) for information on levels.
+
+* `mozilla-group:<groupName>` -
    Roles of this form represent the scopes available to members of the given Mozilla LDAP group via the login service.
 
-* `assume:mozilla-user:<userName>` -
-   Roles of this form represent the scopes available to the given Mozilla LDAP user via the login service.
+* `mozilla-user:<userName>` -
+   Roles of this form represent the scopes available to the given Mozilla LDAP user (email) via the login service.
 
-* `assume:repo:<host>/<path>:branch:<branch>`,
-* `assume:repo:<host>/<path>:pull-request` -
-   Roles of this form represent scopes available to version-control pushes and pull requests.
+* `mozillians-group:<groupName>` -
+   Roles of this form represent the scopes available to members of the given Mozillians group via the login service.
 
-* `assume:moz-tree:level:<level>`
-   Roles of this form include the basic scopes available to version-control trees at each of the three Mozilla source-code managament levels.
-   They are useful as shorthand to configure `repo:*` roles.
-   See [Mozilla Commit Access Policy](https://www.mozilla.org/en-US/about/governance/policies/commit/access-policy/) for information on levels.
+* `mozillians-user:<userName>` -
+   Roles of this form represent the scopes available to the given Mozillians user via the login service.
+
+* `project:<project>:…` -
+   Roles of this form are controlled by the corresponding project.
+
+* `project-member:<project>` -
+   Roles of this form represent the scopes accorded to members of the given project.
+   This role is then be assumed by the appropriate groups.
+   The scopes associated with a `project-member` role are:
```

Comment:
I would think that members of a project should get the scope `assume:project:<project>:*`

I'm not entirely sure how the `project:<project>:...` role-pattern is supposed to be used?
When you say "controlled by the corresponding project" that seems weird...

I maybe wrong... but let's say releng does a special nightly project which is a server that run nightly builds.
I would think the role `project:releng:nigtly-cron` is then given to the client-id that the cron server has, right?

Or do you think that the server would be allowed to programmatically create roles on the form `project:releng:nightly-cron:*` ??